app/assets/stylesheets/templates/_home.scss

```diff
@@ -216,45 +216,110 @@
     }
   }
 
-  .slab--testimonials {
-    background-color: white;
-
+  .slab--rating {
+    padding-bottom: 3.5rem;
+    .title {
+      font-size: 2.7rem;
+    }
+    .text--small {
+      font-size: 1.4rem;
+    }
     #star-rating {
-      margin-bottom: 4rem;
+      img {
+        height: 4rem;
+        width: 100%;
+      }
+      @media screen and (max-width: $tablet-up) {
+        padding: 0;
+      }
     }
-
-    #testimonial-left {
-      padding-right: 6rem;
+    #trusted-support {
+      padding-left: 3rem;
+      @media screen and (max-width: $tablet-up) {
+        text-align: center;
+        padding: 0 1.5rem 0 1.5rem;
+      }
     }
+  }
 
-    #testimonial-right {
-      padding-left: 6rem;
+  .slab-habla-espanol {
+    padding-top: 0;
+    padding-bottom: 0;
+
+    .rounded-box {
+      background-color: #FFD29C;
+
+      a {
+        color: black;
+      }
     }
 
     @media screen and (max-width: $tablet-up) {
-      h2 {
-        padding-left: 2.5rem;
-        padding-right: 2.5rem;
+      .rounded-box {
+        padding: 20px;
       }
+    }
+  }
 
-      #star-rating {
-        margin-top: 2.5rem;
-      }
+  .rounded-box {
+    border-radius: 1rem;
+    padding: 3rem 6rem 3rem 6rem;
+  }
 
+  .slab--testimonials {
+    background-color: white;
+    #testimonial-title {
+      display: flex;
+      flex-direction: column;
+      img {
+        height: 5rem;
+        min-height: 5rem;
+        padding-right: 0;
+        width: 100%;
+      }
+    }
+    #testimonial-left {
+      padding-right: 3rem;
+      .rounded-box {
+        background-color: #FFD29C;
+      }
+    }
+    #testimonial-right {
+      padding-left: 3rem;
+      .rounded-box {
+        background-color: #C2C2FF;
+      }
+    }
+    @media screen and (max-width: $tablet-up) {
```

Comment:
I have a preference for nesting the media query logic into each selector's logic so that it's easier to compare the differences, but that's just my personal preference. It seems like we do a mix of both in the repo.